python/mxnet/gluon/block.py

```diff
@@ -1026,6 +1030,53 @@ def _call_cached_op(self, *args):
             out = [out]
         return _regroup(out, self._out_format)
 
+    def optimize_for(self, x, *args, backend=None, backend_opts=None, **kwargs):
+        """Activates or deactivates :py:class:`HybridBlock` s recursively. Has no effect on
+        non-hybrid children.
```

Comment:
Should this docstring be extended to clarify the difference with `hybridize`?

Reply:
Ya i think i just copied from hybridize's doc string. I'll update this so its more clear what the function does and refer to hybridize. i'll add something like we put in the README.md:
> Immediately partitions a HybridBlock using the specified backend. Combines the work done in the `hybridize` API with part of the work done in the forward pass without calling the CachedOp. Can be used in place of `hybridize`, afterwards `export` can be called or inference can be run. 

What do you think?

Reply:
I think that's reasonable as a separate paragraph in the docstring. But it'd be too confusing as a sentence summary. Maybe for the first sentence, you could describe what optimize_for is used for. Ie. enable users to optimize for a backend without doing an actual forward pass. The concept of CachedOp may not be clear to users.

Reply:
just pushed the change, lmk if that addresses your concern